install.sh

```diff
@@ -317,6 +317,19 @@ EOABORT
 )"
 fi
 
+UNAME_MACHINE="$(uname -m)"
```

Comment:
Can we exclude the whole block when HOMEBREW_ON_LINUX is 1? Just to make sure this Mac-specific code is not run there?

Reply:
@iMichka sorry the page autorefresh didn't show this until after I merged 😬. This should be there still on ARM on Linux because it's an unsupported configuration.

Reply:
Yes it is, but it will tell people to use Rosetta on ARM linux I guess?